security/cmd/istio_ca/main.go

```diff
@@ -48,10 +48,11 @@ import (
 
 type cliOptions struct { // nolint: maligned
 	// Comma separated string containing all listened namespaces
-	listenedNamespaces      string
-	istioCaStorageNamespace string
-	kubeConfigFile          string
-	readSigningCertOnly     bool
+	listenedNamespaces         string
+	unlabeledNamespacesManaged bool
```

Comment:
nit: active var name (e.g., manageUnlabeled) may be a little easier (try both options in an 'if' statement...)

Reply:
changed